src/client/scripts/InitHome.ts

```diff
@@ -10,8 +10,141 @@
+                //If the place returned is the current location, the processing should be different, 
+                //should be picked up from place.i
+                if(place.n===currentLocationString)
+                {
+                    var modifiedCurrentLocation = place.i.replace(",", "/");
+                    placesNearbyUrl= 'https://www.stopbystop.com/placesnearby/'+modifiedCurrentLocation;
+                }
+                else
+                {
+                    placesNearbyUrl= 'https://www.stopbystop.com/placesnearby/'+place.l.a+'/'+place.l.o;
+                }
+                
+                $.ajax({
+                    url : placesNearbyUrl,
+                    dataType: 'json',
+                    method: 'GET',
+                    success : function(result) 
+                    {
+
+                        var imageDiv = $("#Images"), 
+                        myDivs = [],
+                        divIndex = 0,
+                        numOfDivs = result.length;
+                        
+                        if(numOfDivs>10 && window.screen.width>480)
+                        {
+                            numOfDivs=10; //Restrict results to 10 for a desktop screen
+                        }
+                        
+                        if(numOfDivs>6 && window.screen.width<480)
+                        {
+                            numOfDivs=6; //Restrict results to 6 for a mobile screen
+                        }                        
+
+                        for (divIndex; divIndex < numOfDivs; divIndex += 1)
+                        {
+                            var  divId="appendedImagediv"+divIndex;
+                            var  imageId = "appendedImageId"+divIndex;
+                            myDivs.push(InitHome.createDiv(divIndex,result[divIndex]));
+                            
+                            $(myDivs[divIndex]).data('place',result[divIndex]);
+                            $("#Images").append(myDivs[divIndex]);
+                            $(myDivs[divIndex]).on('click', function() {
+                                //if ( $(this).children().length == 0 ) {
+                                    // This is to avoid populating when it already has an image element
+                                    $("#to").val($(this).data('place').n);
+                                    var placeData = { n: $(this).data('place').n, i: $(this).data('place').i };
+                                    $("#to").data('place',placeData);
+                                    $("#view_trip").removeClass("ui-disabled");
+                                //  }
+                             });
+                            var imageElement = document.createElement('img');
+                            imageElement.src = 'https://www.stopbystop.com/client/content/city_images/'+result[divIndex].i+'.jpg';
+                            //imageElement.style.backgroundColor = '#73C5E1';
```

Comment:
same here, let's add it to AppState.current.urls and not reference hardcoded url string